source-mongodb/.snapshots/TestDiscover

```diff
@@ -0,0 +1,83 @@
+Binding 0:
+{
+    "recommended_name": "test/collection1",
+    "resource_config_json": {
+      "database": "test",
+      "collection": "collection1"
+    },
+    "document_schema_json": {
+      "type": "object",
+      "required": [
+        "_id"
+      ],
+      "properties": {
+        "_id": {
+          "type": "string"
+        },
+        "_meta": {
+          "$schema": "http://json-schema.org/draft/2020-12/schema",
+          "$id": "https://github.com/estuary/connectors/source-mongodb/document-metadata",
+          "properties": {
+            "op": {
+              "type": "string",
```

Comment:
Ah seeing this snapshot actually made me think of something else: For other captures with `_meta/op`, it's not actually a `string`, its an `enum` of the possible values, ex: https://github.com/estuary/connectors/blob/654a9467aa5d1790ce66e6e055296b1c28dd8c63/source-postgres/.snapshots/TestDiscoveryCapitalization#L49-L54

Reply:
@williamhbaker added enum to it 👍🏽